packages/odyssey-react/src/components/Modal/index.tsx

```diff
@@ -132,15 +133,13 @@ Modal.Header = ({ children }) => (
         &#8253;
       </Modal.Button>
     </span>
-    <h1 className={styles.title} id="ods-modal-standard-title">
-      {children}
-    </h1>
+    <Title visualLevel="4" children={ children } />
   </header>
 );
 
 Modal.Body = ({ children }) => (
-  <main className={styles.content} id="ods-modal-standard-content">
```

Comment:
we need to avoid the use of static ids like this - even if they are namespaced like this there is no guarantee of uniqueness across the browser frame and that can lead to broken and hard to debug tests, a11y violations, or just broken UI

Reply:
cc @arnoldsandoval-okta can you confirm these ids aren't serving some purpose already?

Reply:
This was an oversight on my part. I can fix it in the next sprint since I'm picking up additional modal functionality. Will log a card.

Reply:
I see what's happening here - referenced by `aria-labelledyby` below. I can fix in this card no worries cc @arnoldsandoval-okta